packages/app/src/app/components/Preview/DevTools/Tests/index.tsx

```diff
@@ -25,24 +23,26 @@ export type Status = 'idle' | 'running' | 'pass' | 'fail';
 
 export type TestError = Error & {
   matcherResult?: boolean;
-  mappedErrors?: Array<{
+  mappedErrors?: {
     fileName: string;
     _originalFunctionName: string;
     _originalColumnNumber: number;
     _originalLineNumber: number;
-    _originalScriptCode: Array<{
-      lineNumber: number;
-      content: string;
-      highlight: boolean;
-    }> | null;
-  }>;
+    _originalScriptCode:
```

Comment:
In this case I prefer using `Array`, so `Array` when using object literals, `[]` for types/interfaces.